### DISCUSSION TITLE
ThrottleRequestsException - Too Many Attempts.

### DISCUSSION THREAD

**Discussion by BenasPaulikas** [pterodactyl/panel#3044] [Support] [answered]
Hey how to increate attempts for pterodactyl panel?

```
{
   "errors": [
       {
           "code": "ThrottleRequestsException",
           "status": "429",
           "detail": "Too Many Attempts."
       }
   ]
}
```

**Comment by trenutoo** [comment, marked as answer]
There are different throttle limits, what are you doing to trigger the throttle limit?

If you are receiving this on your panel/API, then you can adjust the API rate limits by adding and adjusting these parameters in your .env file:

APP_API_CLIENT_RATELIMIT=720
APP_API_APPLICATION_RATELIMIT=240

**Comment by LucaFontanot** [comment]
Also, please remember to do
`php artisan config:clear`

I was wondering why it would not work for me
